src/documents/ImageReordererAPI.md

```diff
@@ -0,0 +1,488 @@
+```
+
+### Parameters ###
+
+#### container ####
+
+The `container` parameter is a CSS-based [selector](http://api.jquery.com/category/selectors/), single-element jQuery object, or DOM element specifying the root node of the Reorderer.
+
+#### options parameter ####
+
+The `options` parameter is an optional collection of name-value pairs that configure the Image Reorderer, as described in the [Options](#options) section below.
+
+## Supported Events ##
+
+The Image Reorderer fires the following events.
+
+<table>
+    <thead>
+        <tr>
+            <th>Event</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Parameters</th>
+            <th>Parameters Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>`onShowKeyboardDropWarning1`</td>
+            <td>default</td>
+            <td>
+                This event fires before a drop warning is displayed.
+            </td>
+            <td>`item`, `kbDropWarning`</td>
+            <td>`item`: The item being moved.
+                `kbDropWarning`: The DOM element that contains the drop warning, and that will be displayed.</td>
+        </tr>
+
+        <tr>
+            <td>`onSelect`</td>
+            <td>default</td>
+            <td>
+                This event fires when an item is selected by the user.
+            </td>
+            <td>`item`</td>
+            <td>`item`: The item being selected.</td>
+        </tr>
+
+        <tr>
+            <td>`onBeginMove`</td>
+            <td>"preventable"</td>
+            <td>
+                This event fires just before a request to move is processed. Because the event is preventable, listeners may prevent the move from happening.
+            </td>
+            <td>`item`</td>
+            <td>`item`: The item being moved.</td>
+        </tr>
+
+        <tr>
+            <td>`onMove`</td>
+            <td>default</td>
+            <td>
+                This event fires just before an item is actually moved.
+            </td>
+            <td>`item`, `requestedPosition`</td>
+            <td>
+                `item`: The item being moved.
+                `requestedPosition`: An object describing the position that the user is trying to move the item into:
+                <pre>
+                    <code>
+requestedPosition = {
+    element,
+    // the drop target
+
+    position
+    // the position, relative to the drop target, that a dragged item should be dropped. One of BEFORE, AFTER, INSIDE, or REPLACE
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+
+        <tr>
+            <td>`afterMove`</td>
+            <td>default</td>
+            <td>
+            This event fires after an item has successfully been moved. For more information, see [Talking to the Server Using The afterMove Event](http://wiki.fluidproject.org/display/docs/Talking+to+the+Server+Using+The+afterMove+Event).
```

Comment:
We should probably create a placeholder file in the new docs for this page, in the to-do folder, and link to that.